libwallet/internal/politeia/politeia.go

```diff
@@ -90,28 +91,45 @@ func (p *Politeia) saveOrOverwiteProposal(proposal *Proposal) error {
 }
 
 // GetProposalsRaw fetches and returns a proposals from the db
-func (p *Politeia) GetProposalsRaw(category int32, offset, limit int32, newestFirst bool) ([]Proposal, error) {
-	return p.getProposalsRaw(category, offset, limit, newestFirst, false)
+func (p *Politeia) GetProposalsRaw(category int32, offset, limit int32, newestFirst bool, key string) ([]Proposal, error) {
+	return p.getProposalsRaw(category, offset, limit, newestFirst, false, key)
 }
 
-func (p *Politeia) getProposalsRaw(category int32, offset, limit int32, newestFirst bool, skipAbandoned bool) ([]Proposal, error) {
+func (p *Politeia) getProposalsRaw(category int32, offset, limit int32, newestFirst bool, skipAbandoned bool, searchKey string) ([]Proposal, error) {
```

Comment:
Please rename `key` and `searchKey` to `searchPhrase` or `searchExpr`. `key` usually carries the meaning of being unique.